RNA-seq/00b-additional_approaches.md

```diff
@@ -23,20 +23,18 @@ That's likely information you'd want to know when performing downstream analyses
 MultiQC supports the three tools we present in the bulk RNA-seq module: FastQC, fastp, and Salmon. 
 In addition, MultiQC is not limited to RNA-seq data; the website has example reports for Hi-C data and whole genome sequencing.
 
-MultiQC is available on the training Docker image.
+MultiQC is installed on the RStudio Server we use for training.
 If you want to run it on the gastric cancer samples, we provide instructions below. 
 Note that you will not have FastQC or fastp output for most samples, so there will be some information missing from the report.
 
 #### Running MultiQC on the gastric cancer samples
 
-Run the training Docker container ([instructions](https://github.com/AlexsLemonade/RNA-Seq-Exercises#post-docker-steps-using-kitematic), set your **LOCAL FOLDER** in Kitematic to the `RNA-seq` folder, and navigate to RStudio in your browser as we did at the beginning of the RNA-seq training module.
-
 Run the following steps in the `Terminal` tab of RStudio.
 
 Set your working directory to the top-level of the RNA-seq folder:
 
 ```bash
-cd kitematic
+cd training-modules/RNA-seq
```

Comment:
Is this the structure we want to use in their home directories? Seems fine, I just need to make it match the actual layout on the server?

Do we want `cd ~/training-modules/RNA-seq` to be sure it always works?

Reply:
The wording above this needs to be changed - good catch

Reply:
I may have fired off that comment too quickly 🤔  - more ☕  required.

> Do we want `cd ~/training-modules/RNA-seq` to be sure it always works?

Yes that's how I envisioned things would be set up - `training-modules` would live in everyone's home directory and then the relevant modules would be in `training-modules` as they are in this directory / analogous to what we have distributed at in-person workshops in the past.

Reply:
See https://github.com/AlexsLemonade/training-modules/pull/193/commits/be8833b297686c05c23b626fc4446258320032a2 - now `cd ~/training-modules/RNA-seq`